tools/codeformat.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python3
```

Comment:
Commit message says `codeformat.sh`, not `.py` :)

Reply:
fixed!